src/assets/styles/theme.scss

```diff
@@ -59,5 +59,6 @@ $BCgovFontColorInverted: #ffffff;
 
 // App Colors
 $app-blue: #1669bb; // same as the Vuetify theme primary
+$p-blue-text: #38598a; // same as the Vuetify theme secondary
```

Comment:
How about a more generic name, like "app-dk-blue"?

Eg: https://github.com/bcgov/namerequest/blob/cd01435b6ddb4f003ebe1ace8593e7ec483c1ef8/client/src/assets/scss/theme.scss#L55

Reply:
Updated the name to `app-dk-blue`